src/ORM/DataObject.php

```diff
@@ -3102,10 +3102,15 @@ public static function get(
      * Return the first item matching the given query.
      * All calls to get_one() are cached.
      *
+     * The filter argument supports parameterised queries (see SQLSelect::addWhere() for syntax examples). Because of
+     * that (and differently from e.g. DataList::filter()) you need to manually escape the field names:
+     * <code>
+     * $member = DataObject::get_one('Member', [ '"FirstName"' => 'John' ]);
+     * </code>
+     *
      * @param string|null  $callerClass The class of objects to be returned. Defaults to the class that calls the method
      *                                  e.g. MyObject::get_one() will return a MyObject
      * @param string|array $filter      A filter to be inserted into the WHERE clause.
-     *                                  Supports parameterised queries. See SQLSelect::addWhere() for syntax examples.
```

Comment:
Did you mean to remove this line? It's still relevant?

Reply:
Yes to both ;) I just moved it in the body to be able to append paragraph and example mentioning the quoting issue.